TechTable/tt_table.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html>
+ <head>
+  <title>Courses offered at IIIT</title>
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.0/jquery.min.js"></script>
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css" />
+  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
+ </head>
+ <body>
+  <div class="container">
+   <div class="table-responsive">
+    <h1 align="center">Courses offered at IIIT</h1>
+    <br />
+    <div align="center">
+     <button type="button" name="load_courses" id="load_courses" class="btn btn-info">Load Courses</button>
+    </div>
+    <br />
+    <div id="courses_table">
+    </div>
+   </div>
+  </div>
+ </body>
+</html>
+
+<script>
+$(document).ready(function(){
+ $('#load_courses').click(function(){
+  $.ajax({
+   url:"Courses.csv",
+   dataType:"text",
+   success:function(data)
+   {
+    var courses_data = data.split(/\r?\n|\r/);
+    var table_data = '<table class="table table-bordered table-striped">';

```

Comment:
Instead of preparing the table as string, you should [prepare it using jQuery](https://stackoverflow.com/questions/268490/jquery-document-createelement-equivalent)

The reason behind this is, what if some course name in the future is weird and it looks like "</td>"? That would mess up your entire table. Similarly, there are a number of inputs that can mess up your table. When you use strings, there's no distinction between "</td>" as html and "</td>" in a a course title.

Reply:
This block of code is from the older version of the file- for the Course Table which has now been moved to the folder for old files

Reply:
In the current version of the table, the CSV file data is being converted to a php array, which is subsequently converted to a json string to be used as a data source by DataTables.